web/app/css/octopus.css

```diff
@@ -1,30 +1,33 @@
 @import 'styles.css';
 
 .octopus-graph {
-  padding: calc(var(--base-width) * 4);
+  max-width: 940px;
```

Comment:
Nit: I'm thinking we could probably add a little bit of padding to the edge of the octopus graphs. Something like:

```css
.octopus-graph {
    max-width: 980px;
    padding: 15px 20px;
    ...
}
```